getUsers() for api version 2

Just-Web-Developer opened the pull request:
Added compatiblity of function "getUsers" with api version 2

Just-Web-Developer committed:
Update jira.js

Fix of function "getUsers" for api version 2

Seth10001 commented on src/jira.js:
```diff
@@ -829,10 +829,11 @@ export default class JiraApi {
   getUsers(startAt = 0, maxResults = 100) {
     return this.doRequest(this.makeRequestHeader(this.makeUri({
-      pathname: '/users',
+      pathname: this.apiVersion === '2' ? '/user/search' : '/users',
```
`/rest/api/2/user/searh` appears to be an alias for `/rest/api/2/users`: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-users/#api-rest-api-2-users-get

This change shouldn't be needed? Could you provide some more context?

Just-Web-Developer replied:
`/rest/api/2/users` doesn't worked for a local copy of Jira, but `/rest/api/2/user/searh` worked well. Now, I checked it with cloud Jira and got the opposite result, `/rest/api/2/user/searh` doesn't work but `/rest/api/2/users` do. Maybe it would be usefull to add an additional parameter that would indicate whether the request sent to a local Jira or not.

Just-Web-Developer committed:
Update jira.js

Added "local" an additional parameter that would indicate whether the request sent to a local Jira or not.
